submissions/kolawoleOgunfowokan/PHPTask/php/create.php

```diff
@@ -0,0 +1,50 @@
+<?php
+session_start();
+require "helper.php";
+require "fetch.php";
+
+if (isset($_POST['submit'])) {
+    try {
+        $connection = new PDO($dsn, $username, $password, $options);
+
+        $newProduct = array(
+            "itemname" => $_POST['itemname'],
+            "quantity" => $_POST['quantity'],
+            "brand_id" => $_POST['brand']
+        );
+        /**
+         * This is equivalent to
+         * INSERT INTO products (itemname, quantity) values (:itemname, :quantity)
+         */
+        $sql = sprintf(
+            "INSERT INTO %s (%s) values (%s)",
+            "products",
+            implode(", ", array_keys($newProduct)),
+            ":" . implode(", :", array_keys($newProduct))
+        );
+
+        $statement = $connection->prepare($sql);
+        $statement->execute($newProduct);
+    } catch (PDOException $error) {
+        echo $sql . "<br>" . $error->getMessage();
+    }
+}
+?>
+<?php if (isset($_POST['submit']) && $statement) { ?>
+    > <?php echo $_POST['itemname']; ?> successfully added.
+<?php } ?>
+<h2>Add A Product </h2>
+<form method="post">
+    <label for="itemname">Name of Product</label>
+    <input type="text" name="itemname" id="itemname">
+    <select name="brand">
+        <?php foreach ($_SESSION['brands'] as $key => $value) { ?>
```

Comment:
You should not use Sessions. You can always query your database when you are trying to render your form.